salalib/pointdata.cpp

```diff
@@ -431,7 +430,9 @@ bool PointMap::clearPoints()
                m_points[i][j].set( Point::EMPTY, m_undocounter );
                if (!m_points[i][j].m_merge.empty()) {
                   PixelRef p = m_points[i][j].m_merge;
-                  m_merge_lines.remove_at(m_merge_lines.searchindex(PixelRefPair(PixelRef(i,j),p)));
+                  auto it = std::find(m_merge_lines.begin(), m_merge_lines.end(), PixelRefPair(PixelRef(i,j),p));

```

Comment:
Probably worthwhile putting this in a helper function (templated) - it will appear all over the shop